wicamo96/FieldScout
# House bays

# Description

Added functionality to Greenhouse Management route which allows you to CRUD bays and bay divisions.

## Type of change


# Testing Instructions

### Create database and populate data if necessary
- If you haven't created the FieldScout database, please do so in visual studio community.
- Use this to fill out tables and columns: [SQL data](https://github.com/wicamo96/FieldScout/blob/main/SQL/01_FieldScout_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/wicamo96/FieldScout/blob/main/SQL/02_FieldScout_Seed_Data.sql)

### Testing Greenhouse Management CRUD
- Click the execute with debugger solid green button to launch the backend in visual studio community.
- From the client directory in a git terminal, run `npm run dev`.
- Once the Vite + React window pops up, you should be directed to the log in page.
- Log in with the default email.
- Navigate to Greenhouse Management via the link in the navbar.
- You should see a list containing all the houses associated with your facility. (Default is House A, and House B).
- Click House A to be directed to a page with all the bays associated with that house (Default is Bay 1).
- Click add new bay and enter a test name, submit and confirm its addition to the list.
- Test out the edit and delete features for your test bay.
- Click Bay 1 to be directed to a page with all the bay divisions associated with that bay (Default is Division A and Division B).
- Click add new division and enter a test name, submit and confirm its addition to the list.
- Test out the edit and delete features for your test bay division.

# Checklist:
